packages/opentelemetry-instrumentation-langchain/tests/test_langgraph.py

```
@@ -136,21 +134,17 @@ def build_graph():
     assert trace.get_current_span() == INVALID_SPAN
 
     spans = span_exporter.get_finished_spans()
-    assert [
-        "mynode.task",
-        "LangGraph.workflow",
-    ] == [span.name for span in spans]
+    assert [span.name for span in spans] == ["mynode.task",
+        "LangGraph.workflow",]
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)